vllm/model_executor/models/qwen2_vl.py

```diff
@@ -982,7 +984,7 @@ def __init__(self,
         self.visual = Qwen2VisionTransformer(
             config.vision_config,
             norm_eps=getattr(config, "rms_norm_eps", 1e-6),
-            quant_config=quant_config,
+            quant_config=self._maybe_ignore_quant_config(quant_config),
```

Comment:
I have a question: What should we do if our users execute their own quantization, and they have quantized the visual encoder?